roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -225,6 +225,7 @@ kiali_defaults:
     web_fqdn: ""
     web_root: ""
     web_schema: ""
+    web_history_mode: ""
```

Comment:
As per [coding standards](https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting), place this in alphabetical order within this group of settings - so `web_history_mode` should be placed before `web_root` and after `web_fqdn`.